addons/website/static/src/scss/website.scss

```diff
@@ -774,3 +793,29 @@ table.table_desc tr td {
         background-repeat: no-repeat;
     }
 }
+
+// Scroll down button
+.o_scroll_button {
+    border: none;
+    font-size: 3em;    
+    animation: o-anim-pulse 1s cubic-bezier(1,0,.8,1) 1s infinite alternate;
+    width: 50px;
+    height: 50px;
+    border-radius: 50%;
+
+    &:focus {
+        outline: none;
+    }
+}
+
+// Hide button when full-height is not enabled.
+section:not(.o_full_screen_height) .o_scroll_button {
+    display: none !important;
+}
+
+@keyframes o-anim-pulse {
+    100% {
+        transform: scale(1.1);
+    }
+}
```

Comment:
Is not there a style in blog for this already ? So that we may share it ?

Reply:
Initially, this was done so that the animation color could be customized (can only animate to inline style value by leaving 0% untouched in the animation) but in the end @sbuhl didn't think it was necessary so I can probably just share most of the code.

Reply:
Let's share then :)